test/java/org/chromium/net/testing/CronetTestRule.java

```diff
@@ -101,9 +101,12 @@ public Statement apply(final Statement base, final Description desc) {
     return new Statement() {
       @Override
       public void evaluate() throws Throwable {
-        setUp();
-        runBase(base, desc);
-        tearDown();
+        try {
+          setUp();
+          runBase(base, desc);
+        } finally {
+          tearDown();
```

Comment:
Good catch - by contract, tearDown() is to be called in any circumstance. Thanks.

Reply:
You're welcome.